frontend/src/logger/pcm.logger.ts

```diff
@@ -0,0 +1,75 @@
+import {AxiosInstance} from 'axios'
+import {AppConfig} from '../app-config/types'
+import {handleNotAuthorizedErrors} from '../auth/handleNotAuthorizedErrors'
+import identityFn from 'lodash/identity'
+
+enum LogLevel {
+  info = 'INFO',
+  warning = 'WARNING',
+  debug = 'DEBUG',
+  error = 'ERROR',
+  critical = 'CRITICAL',
+}
+
+interface LogEntry {
+  message: string
+  level: LogLevel
+  extra?: {[key: string]: string}
+}
+
+export class PcmLogger {
```

Comment:
suggestion: I wouldn't use "PCM" in the namings, it's just a Logger pretty much as "Home" is not called "PCMHome"